### ISSUE TITLE
Default effort level

### ISSUE THREAD

**Issue by kliberty** [FLIF-hub/FLIF#510]
The program help says effort of `-E 60` is the default, but some cursory tests I believe `10 - 14` may be a better default.

I attached the image I used, as well as a plot showing the size-png_size ratio and time-max_time ratio vs. the effort value. I tried a couple other images with similar results, though, I did not attempt to cover types of image data.

I don't mind running it for other images, [our you can use the scripts I used](https://gist.github.com/kliberty/f000119f49ff61eec46efc8982d4cc13).

![flifratios](https://user-images.githubusercontent.com/8992170/40870383-352d8846-65fc-11e8-8c0a-9ecc56c7e2fc.png)
![ash_tree_-_geograph org uk_-_590710](https://user-images.githubusercontent.com/8992170/40870382-2bd62b54-65fc-11e8-89d4-6e86bed73bcf.png)

**Comment by lehitoskin** [comment]
Yeah, that might be fine. The more data that is available, the better, IMHO.

**Comment by jonsneyers** [comment]
I think `-E30` is probably a better default.

**Comment by kliberty** [comment]
Why 30? 
I'm running all the images in `flif-benchmarks/test_images` right, but I can only let it run while I'm sleeping and since I tweaked it for better timing statistics it is taking longer than I expected..

So far I've completed :
  * debian
  * icip-corel
  * Lukas-2D-8bit
  * openstreetmap
  * png-gallery
  * pngimg.com-car-logos
  * pngimg.com-hammers

**Comment by kliberty** [comment]
A website seemed like the best way to display the results. They're posted at [https://kliberty.bitbucket.io/flif-bench/](https://kliberty.bitbucket.io/flif-bench/).

I few images a missing because of some unrelated computer issues...but there are still more than 900 tests available.
